shared/schemas/workflow-status-schema.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowStatus",
+    "description": "Status of workflow execution",
+    "type": "object",
+    "properties": {
+        "status": {
+            "description": "The current status of a workflow execution",
+            "type": "string",
+            "enum": [
+                "running",
```

Comment:
Any other statuses?

The `running` is for not-yet finished workflows. 
What about introducing progress-info here? To get sort of info `Finished 3 of 6 tasks` but in a structured way.

Reply:
there are cases where it is not easy to say how many tasks are done.

Reply:
these are the statuses coming from sonata:
ACTIVE, ERROR, COMPLETED, ABORTED, SUSPENDED, PENDING
The status field is populated by sonata, that's why I'm not sure it's a good idea to duplicate it the new status field